packages/components/src/stories/decorators/ThemeDecorator.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
This is amazing! I didn't know it was possible to wrap each story in a themewrapper on it's own ✨